user_list.csv

```diff
@@ -0,0 +1,98 @@
+a74d7f82-ef96-4195-a3e3-b16d1b89f4a2,True,False,Field Workers,Solwezi District TBA
+e090fa75-42e8-461e-89a0-be504c61b4b3,True,False,Field Workers,Chikankata District TBA
+a3edee17-74a4-4ce3-93c4-a529dfce026e,True,False,Field Workers,Chikankata District TBA
+287af02b-f66c-42f0-9bfe-17107bbed1e7,True,False,Field Workers,Choma District TBA
+929f0b01-61ef-439d-b3f9-dd45ae4cfd17,True,False,Field Workers,Choma District TBA
+c4041336-3f89-4a77-a20a-cab969dcd720,True,False,Field Workers,Choma General Hospital
+3988a973-0929-4f4c-aac5-b7c90256b8c7,True,False,Field Workers,Choma General Hospital
+06cd6eba-074f-4bef-b009-8eb58ff3f9c5,True,False,Field Workers,Kalomo Urban Clinic
+19c05fc6-96de-43a9-bb56-398b7d2629fe,True,False,Field Workers,Kalomo Urban Clinic
+d8619fa5-3bd6-4493-b4b4-26d9d04e5dfc,True,False,Field Workers,Request Muntanga District Hospital
+0accced4-a071-4fe1-8e96-434ba688d4a9,True,False,Field Workers,Request Muntanga District Hospital
+d2ed9d00-2651-431c-a915-9743f3cb099d,True,False,Field Workers,Livingstone Central Hospital
+22da7b32-3aa6-44c3-b382-9cf92a5dc77f,True,False,Field Workers,Livingstone Central Hospital
+3e32b460-11cb-4b3b-bb08-d2f4cb68bfa0,True,False,Field Workers,Mazabuka General Hospital
+d9f9cbd8-1c98-4fd2-bf7d-5839cd4e1998,True,False,Field Workers,Mazabuka General Hospital
+c2d1fe2f-bafe-492d-94ee-3410a4707361,True,False,Field Workers,Chikuni District TBA
+ab215e8d-eea2-4213-b71b-6e90e65f5094,True,False,Field Workers,Chikuni District TBA
+c1a9a45a-55af-4e99-98f9-2ba13a6dcbb5,True,False,Field Workers,Monze Mission Hospital
+7db575a4-0080-4de6-9366-27831da7cb19,True,False,Field Workers,Monze Mission Hospital
+c79fbd53-9316-4368-a791-037a5a60460b,True,False,Field Workers,Namwala District Hospital
+4ea3b78d-b394-4f8f-9d73-b3db626a63ef,True,False,Field Workers,Namwala District Hospital
+a9ea057d-4f82-4770-aa9f-90670b3bcb3c,True,False,Field Workers,Siavonga District TBA
+aeba7258-4fe6-4c84-a24e-8514dc8dac9d,True,False,Field Workers,Siavonga District TBA
+388b0905-5ab9-4368-894a-22a918c7ec35,True,False,Field Workers,Maamba Hospital
+ca4e0818-3cba-4f51-90a9-d4f94cc15a3c,True,False,Field Workers,Maamba Hospital
+8a1c12d6-47aa-404f-bbb0-c4c69f39babc,True,False,Field Workers,Kaoma District Hospital
+7aa98c10-86fb-40d7-bb86-da06979d341d,True,False,Field Workers,Kaoma District Hospital
+aa40c270-6647-4502-ba10-b00074b7dafc,True,False,Field Workers,Lewanika General Hospital
+4f57804b-3fe7-4dd7-ae60-8f0a747c3dd4,True,False,Field Workers,Lewanika General Hospital
+967c897d-672a-4483-be97-168448a3c4c5,True,False,Field Workers,Senanga General Hospital
+74a67d2b-279f-480a-812f-5b3cf57aa15d,True,False,Field Workers,Senanga General Hospital
+548414fb-137c-4582-bf13-332142caa786,True,False,Field Workers,Mwandi Mission Hospital
+656ef15f-4df9-43df-9f2f-c34e88d7fd63,True,False,Field Workers,Mwandi Mission Hospital
+5451b5c5-e910-48cb-ad67-660c7bd7fbad,True,False,Data Viewers,
+f239287c-eb0e-4d03-b337-8ed5009bdd07,True,False,,
```

Comment:
Should this file be tracked?

Reply:
ah no, sorry forgot to add this to .gitignore